administration/package.json

```diff
@@ -2,69 +2,101 @@
   "name": "administration",
   "version": "3.1.2",
   "private": true,
+  "dependenciesComments": {
+    "typescript": "Keeping on 5.0.x because of current incompatibility of 5.1.x with @typescript-eslint"
+  },
   "dependencies": {
-    "@apollo/client": "^3.7.9",
-    "@blueprintjs/core": "^4.16.3",
-    "@blueprintjs/popover2": "^1.13.3",
-    "@blueprintjs/select": "^4.9.3",
-    "@blueprintjs/table": "^4.9.3",
-    "@bufbuild/protobuf": "^1.0.0",
-    "@fontsource/roboto": "^4.5.8",
-    "@mui/icons-material": "^5.11.9",
-    "@mui/material": "^5.11.10",
-    "@zxing/library": "^0.19.2",
+    "@apollo/client": "^3.7.16",
+    "@blueprintjs/core": "^5.0.0",
```

Comment:
[Also, if I remember correctly, removing the dependency will result in a compilation error.]